src/main/features/offer/routes/defendant-response.ts

```diff
@@ -0,0 +1,44 @@
+import * as express from 'express'
+import { FormValidator } from 'forms/validation/formValidator'
+import { Paths } from 'offer/paths'
+import { Form } from 'forms/form'
+import { DefendantResponse } from 'offer/form/models/defendantResponse'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+import { OfferGuard } from 'offer/guards/offerGuard'
+
+function getOffer (claim: any): any {
+  if (!claim.settlement || !claim.settlement.partyStatements) {
+    return undefined
+  }
+  return claim.settlement.partyStatements[0].offer
```

Comment:
This should return last offer not the first one since the page will be reused for responding to more then one defendant offer. Implement something similar to `uk.gov.hmcts.cmc.claimstore.models.offers.Settlement#getLastStatement`